wasmi-impl/src/lib.rs

```diff
@@ -24,7 +24,7 @@ pub fn exec_wasm_with_data(
     let module = wasmi::Module::from_buffer(binary)?;
 
     // TODO: Calculate the memory size always be larger than `data`
-    let mem_instance = MemoryInstance::alloc(Pages(200), None)?;
+    let mem_instance = MemoryInstance::alloc(Pages(13000), None)?;
```

Comment:
Is 13000 an arbitrary number?

Reply:
Kind of, it was the maximum size I could allocate without errors.

It is 13000*64 KiB memory, which is ~850 MB

The theoretical limit is 65536, (4GB) but I cannot allocate enough memory on my VM

We need to test it with more memory allocated in [Enclave.config.xml](https://github.com/ntls-io/wasm-exec-sgx/blob/main/enclave/Enclave.config.xml)

Reply:
Do you have a link to where the theoretical limit comes from?

A single unit of Pages is 65536 bytes I believe - https://docs.rs/memory_units/latest/memory_units/wasm32/struct.Pages.html.

Reply:
Sorry here you go: https://paritytech.github.io/wasmi/wasmi/struct.MemoryInstance.html

Reply:
Ok cool thanks.